src/components/query-selector/export-type.tsx

```diff
@@ -0,0 +1,47 @@
+'use client';
+import { Center, GridCol, Paper, rem, Select, Stack, Title } from '@mantine/core';
+import { useState } from 'react';
+import BuildQueryButton from './build-query-button';
+import { capitalize } from 'lodash';
+
+export type SupportedExportTypes = 'patient' | 'group' | 'system';
+
+export interface ExportTypeProps {
+  exportType: SupportedExportTypes;
+  dropdownData?: string[];
+}
+
+/*
+ * Component for selecting which type of export the user wants.
+ */
+export default function ExportType({ exportType, dropdownData }: ExportTypeProps) {
+  const [queryId, setQueryId] = useState<string | null>('');
```

Comment:
I think we either want to just make this type a string or initialize it as null? https://gist.github.com/mgramigna/bdc933fae2150216681c28d1f0f7aac7#hooks